--- origsrc/goffice-0.7.14/configure.in	2009-09-27 15:45:50.000000000 -0500
+++ src/goffice-0.7.14/configure.in	2009-10-28 21:33:09.609780400 -0500
@@ -230,7 +230,7 @@ with_native_win32=no
 case $host_os in
   mingw* | pw32* | cygwin*)
     with_win32=yes
-    GOFFICE_PLUGIN_LDFLAGS="-Wl,--enable-runtime-pseudo-relo,--export-all-symbols $GOFFICE_PLUGIN_LDFLAGS"
+    GOFFICE_PLUGIN_LDFLAGS="-Wl,--export-all-symbols $GOFFICE_PLUGIN_LDFLAGS"
     ;;
 esac
 AC_MSG_RESULT($with_win32)
